test/boost/sstable_compaction_test.cc

```diff
@@ -5017,3 +5017,69 @@ SEASTAR_TEST_CASE(compaction_manager_stop_and_drain_race_test) {
     testlog.info("stopping compaction manager");
     co_await cm.stop();
 }
+
+// dht::subtract_ranges is used by cleanup_compaction
+// get_ranges_for_invalidation.
+SEASTAR_THREAD_TEST_CASE(test_dht_subtract_ranges) {
+    simple_schema ss;
```

Comment:
it can be moved to partitioned_test or range_test.

Reply:
I wanted to place it in range_test, but it is a a pure boost test and subtract_ranges is a coroutine.
partitioner_test looks good.